config/locales/en.yml

```diff
@@ -803,7 +803,7 @@ en:
           email:
             label: "Email"
         contact_recipient:
-          label: "Which of the following best describes your inquiry?"
+          label: "Whom do you want to contact?"
```

Comment:
Not an English native, but I think it should be "who" instead of "whom"

Reply:
Okay changed.